client/python/datajunction/__init__.py

```diff
@@ -3,16 +3,17 @@
 """
 from importlib.metadata import PackageNotFoundError, version  # pragma: no cover
 
-from datajunction.client import (
+from datajunction._internal import (
```

Comment:
The way this import is will unhide all of the internal classes here (like in the below example). It's probably a pain to underscore all of the classes so I wonder if just doing `from datajunction import _internal` and then using `_internal.Cube` in this file will keep them hidden.
```py
import datajunction
for i in dir(datajunction):
  print(i)
```
_output_
```
AvailabilityState
ColumnAttribute
Cube
DIST_NAME
DJReader
DJWriter
Dimension
Engine
MaterializationConfig
Metric
Namespace
Node
NodeMode
Source
Transform
_DJClient
__all__
__builtins__
__cached__
__doc__
__file__
__loader__
__name__
__package__
__path__
__spec__
__version__
_internal
client
exceptions
models
```

Reply:
Yeah, I haven't gotten to this point yet. I only cleaned up the names I moved to the new client.py so far. But I like your suggestion on `from datajunction import _internal`. Will consider that!